src/planning/plan/tests/plan-consumer-test.ts

```diff
@@ -41,6 +41,7 @@ async function storeResults(consumer: PlanConsumer, suggestions: Suggestion[]) {
 
 describe('plan consumer', () => {
   it('consumes', async () => {
+    DriverFactory.clearRegistrationsForTesting();
```

Comment:
Are there default registrations that we're removing here? Just haven't come across this before.

Reply:
something is installing registrations in a before() or beforeEach() handler and it's infecting other test files, I think. At the moment I'm just adding these when I find a unit test that fails when it runs in isolation.

Reply:
Seems like a decent thing to do safety wise.

Could it be the beforeEach block of 'Direct Store Muxer'?
There's no afterEach to clear our the register so I think the last test will leave a MockStorageDriverProvider registered.